processor/htmlproc/htmlproc_test.go

```diff
@@ -39,27 +39,62 @@ func makeResponse(url, html string) *exchange.Response {
 }
 
 func TestHTMLProcessor_Default(t *testing.T) {
```

Comment:
rename the test name?

Reply:
I thought it was more or less "default," but done.